### DISCUSSION TITLE
what are the differences between the esp32par*, esp32par*Q and esp32par*QQ

### DISCUSSION THREAD

**Discussion by nielsnl68** [moononournation/Arduino_GFX#231] [Q&A] [answered]
Hi, 
i'm trying to understand what those 3 class groups are being used for.
Comparing them i see some code differences but i dont know which to use atm.

**Comment by moononournation** [comment, marked as answer]
The basic class is most compatible, and more Q should be faster. Once you tested basic class ok, you can try faster class.
